extensions/ql-vscode/src/databases/github-nwo.ts

```diff
@@ -28,7 +28,10 @@ export function looksLikeGithubRepo(
  * @param githubUrl The GitHub repository URL
  * @return The corresponding NWO, or undefined if the URL is not valid
  */
-export function convertGitHubUrlToNwo(githubUrl: string): string | undefined {
+export function convertGitHubUrlToIdentifier(
```

Comment:
Is it a good idea to let this method handle both case? We would have some duplication if we create two methods.. Like this we will have to put it in a different file or rename the file though.

Reply:
I don't have any strong opinions either way 😅 I think this is okay, if you could update function description and return value above? 

Also, this isn't something you added in this PR, but the name "convert ..." feels a bit unclear. Perhaps we could rename this function to something like "get owner (or identifier) from URL"?